### ISSUE TITLE
Post-Manubot update cleanup

### ISSUE THREAD

**Issue by agitter** [greenelab/deep-review#700]
We wanted to merge #681 quickly, so here are some lingering questions and confirmations from that merge:
- [ ] Inspect pdf formatting and references in detail
- [ ] Ensure that all author info transferred correctly
- [ ] Customize the front matter to reduce the length of the author list
- [ ] Add back footnotes for corresponding authors and the randomized author order
- [ ] Replace contribution, funding, and other templates

@dhimmel if you are still willing to re-engineer the author template scripts, that will address most of these.  I'll review the formatting and author info that I didn't scrutinize during the #681 review.

**Comment by agitter** [comment]
While reviewing the author information I noticed the unnecessary `OTHER` in Yifan Peng's funding information.  I'll leave it until we decide what we're going with the templates.

https://doi.org/10.1371/journal.pbio.1001661 isn't working with the new CSL.  The last author is not show, perhaps because it is a consortium.  The rest of the pdf formatting and reference list looked good.

**Comment by agitter** [comment]
@dhimmel do you still plan to work on the new templating that will replace the author table processing functions?  We can at least decide which templates that were lost in #681 should be re-implemented.

**Comment by dhimmel** [comment]
> do you still plan to work on the new templating that will replace the author table processing functions?

Yeah. It seems like that the author section is the only part that still needs reworking? I could make it like the format from [preprint v1](https://www.biorxiv.org/content/early/2017/05/28/142760.full.pdf), if that's what you prefer. Let me know if there are any improvements on that format you'd like before I get started.

**Comment by agitter** [comment]
I believe daef821 is the last time I added templates.  There I see templates in the [front matter](https://github.com/greenelab/deep-review/blob/daef821cc87f264063cb0f32e79d284a756d56bc/sections/00_title_authors.md) for listing authors, affiliations, and corresponding authors and the [methods](https://github.com/greenelab/deep-review/blob/daef821cc87f264063cb0f32e79d284a756d56bc/sections/08_methods.md) for funding.  I thought I also had them for the contributions statement, but apparently I didn't implement that.

With so many authors, I do prefer having the front matter author list be more compact.  It wouldn't need to be identical to `preprint v1`, but something that puts all authors in a single wrapped line followed by a list of affiliations will fit better on a single PDF page.  We could make the affiliations smaller font to further save space.  The notes about author order and corresponding authors could be hard coded because those don't change.  We do have some repeated affiliations.

What do you think about the funding section?  That will need to be updated after we add the new v2 authors.  We can use the `metadata.yaml` support for initials to add those manually instead of deriving them from names like I did previously.

We also need to update the contributions statement, but I don't see a good way to template that with our current `metadata.yaml` fields.

**Comment by dhimmel** [comment]
> What do you think about the funding section?

Let's do the funding section manually.

> We also need to update the contributions statement

Let's also do this manually.

The automated section would be the author's list and affiliations.

**Comment by agitter** [comment]
@dhimmel do you agree we can close this per #769 and #771?  The contributions and funding will be updated manually.  I already have a reminder in #561.
